src/components/notifications/AppSelector/index.tsx

```diff
@@ -124,47 +130,59 @@ const AppSelector: React.FC = () => {
           )}
         </div>
         <div className="AppSelector__wrapper">
-          <Label color="main">Subscribed</Label>
+          {!loading && filteredApps?.length > 0 ? <Label color="main">Subscribed</Label> : null}
           <ul className="AppSelector__list">
             {loading
               ? Array(3)
                   .fill(<LinkItemSkeleton />)
                   .map(x => x)
               : null}
             {!loading &&
-              filteredApps?.map(app => (
-                <AnimatePresence>
-                  <m.div initial={{ opacity: 0 }} exit={{ opacity: 0 }} animate={{ opacity: 1 }}>
-                    <NavLink
-                      key={app.topic}
-                      to={`/notifications/${app.topic}`}
-                      className="AppSelector__link-item"
-                    >
-                      <div className="AppSelector__notifications">
-                        <div className="AppSelector__notifications-link">
-                          <div className="AppSelector__link-logo">
-                            <img
-                              src={app.metadata.icons?.[0] || '/fallback.svg'}
-                              alt={`${app.metadata.name} logo`}
-                              onError={handleImageFallback}
-                              loading="lazy"
-                            />
-                          </div>
-
-                          <div className="AppSelector__link__wrapper">
-                            <Text className="AppSelector__link__title" variant="small-500">
-                              {app.metadata.name}
-                            </Text>
-                            <Text className="AppSelector__link__subtitle" variant="small-500">
-                              {app.metadata.description}
-                            </Text>
+              filteredApps?.map(app => {
+                const unverified = isUnVerified(app, projects)
```

Comment:
Nit pick: prefix the variable with `is` to denote it's a boolean and rename the function.
```suggestion
                const isUnverified = checkIsUnVerified(app, projects)
```